### DISCUSSION TITLE
Zen Browser for Android and iOS devices

### DISCUSSION THREAD

**Discussion by ArskaBoii** [zen-browser/desktop#3853] [Polls]
Zen Browser needs their mobile app too for Android and iOS!!

**Comment by JSGRD22** [comment]
How would you design the UI? The arc developers have already tried carrying the design language over to a mobile app and failed.

**Comment by giaaaacomo** [comment]
the desktop app is in beta, i'd prefer the devs to focus on that. Also building an app from scratch takes time; overall the app is secondary for me

**Comment by syscat29** [comment]
This is something that shouldn't even be thought of until the browser is fairly stable, also developing for mobile devices is a whole new thing, Im closing this discussion not really doing anything.
